test/testcne/metrics_test.c

```diff
@@ -0,0 +1,114 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright (c) 2022 Intel Corporation
+ */
+
+// IWYU pragma: no_include <bits/getopt_core.h>
+
+#include <stdio.h>           // for NULL, EOF
+#include <metrics.h>         // for metrics_register
+#include <tst_info.h>        // for tst_error, tst_end, tst_start, TST_FAILED
+#include <getopt.h>          // for getopt_long, option
+
+#include "metrics_test.h"
+
+static int
+test_cb(metrics_client_t *client, const char *cmd __cne_unused, const char *params __cne_unused)
+{
+    printf("Test Callback\n");
```

Comment:
Can you please use cne_printf() instead of printf?

Reply:
Done